### ISSUE TITLE
Project creation is broken in version 3

### ISSUE THREAD

**Issue by mplis** [nozzle/react-static#86]
Running `react-static create` with version 3.0.1 in a directory called `test` results in a file structure like this:
```
test
  my-static-site/
    package.json
    public/
    src/
    static.config.js
  node_modules/
  package.json
  yarn.lock
```

test/package.json:
```
{
  "dependencies": {
    "react-static": "^3.0.1"
  }
}
```

test/my-static-site/package.json:

```
{
  "name": "react-static-example-basic",
  "version": "1.0.1",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "start": "react-static start",
    "build": "react-static build",
    "serve": "serve dist -p 3000"
  },
  "dependencies": {
    "axios": "^0.16.2",
    "react": "^16.0.0",
    "react-dom": "^16.0.0",
    "react-router": "^4.2.0"
  },
  "devDependencies": {
    "eslint-config-react-tools": "1.x.x",
    "serve": "^6.1.0"
  }
}
```

Trying to run `npm run build` inside `test/my-static-site` results in a bunch of errors, presumably because the dependencies are all messed up.

**Comment by mplis** [comment]
I think I figured it out. Upgrading node from `v6.9.1` to `v8.3.0` fixed the problem. It might have had something to do with [this](https://github.com/nozzle/react-static/blob/f598a8cb6680105d28ba23f39f240c9f59c8912f/src/commands/create.js#L57) line not doing what's expected in `v6.9.1`.

**Comment by tannerlinsley** [comment]
This was absolutely a bug. Fixed in 3.0.3
